AUTHORS.md

```diff
@@ -0,0 +1,6 @@
+Contributors in the order of first contribution
+
+- [Sudar Muthu](https://github.com/sudar)
+- [msdfgrant](https://github.com/msdfgrant)
+- [Hughie Wilmshurst](https://github.com/HughieW)
```

Comment:
You have missed two more people.

https://github.com/sudar/bulk-move/pull/4
https://github.com/sudar/bulk-move/pull/3

(I am not sure why github is not picking up there names through)